docker/mongodb-kubernetes-tests/tests/tls/e2e_tls_disable_and_scale_up.py

```diff
@@ -38,18 +40,11 @@ def test_rs_is_running(replica_set: MongoDB):
 def test_tls_is_disabled_and_scaled_up(replica_set: MongoDB):
     replica_set.load()
     replica_set["spec"]["members"] = 5
```

Comment:
The issue in the test is that it was doing the update in two steps (scale, and then disable tls), while the whole point is to change them at the same time.